src/string/lipogram.rs

```diff
@@ -0,0 +1,78 @@
+use std::collections::HashSet;
+
+//*
+// Fn that returns the letters that are missing from the input slice
+// and are present in the english alphabet
+fn compute_missing(in_str: &str) -> HashSet<char> {
+    let alphabet: HashSet<char> = "abcdefghijklmnopqrstuvwxyz".chars().collect();
+
+    let letters_used: HashSet<char> = in_str
+        .to_lowercase()
+        .chars()
+        .filter(|c| c.is_ascii_alphabetic())
+        .collect();
+
+    alphabet.difference(&letters_used).cloned().collect()
+}
+
+//*
+// Fn that checks if the slice is a lipogram.
+// Lipogram - sentence in which a particular letter or group of letters is avoided
+pub fn is_lipogram(lipogram_str: &str) -> bool {
```

Comment:
I am more and more convinced that one should specify the _target missing characters_. For example:
`"abcdefghijklmnopqrstuvwx"` is a `{'y', 'z'}-lipogram` but not `{'z'}-lipogram`.

I would suggest to change the signature of this function into:
```suggestion
pub fn is_lipogram(lipogram_str: &str, missing_chars: &HashSet<char>) -> bool {
```